app/shared/api/blockchain/transfer/types.ts

```diff
@@ -0,0 +1,22 @@
+import type { KeyringPair } from '@polkadot/keyring/types';
+import { type Hash } from '@polkadot/types/interfaces';
+import type { BN } from '@polkadot/util';
+
+export interface ITransfer {
+  sendTransfer: (params: SendTransferParams) => Promise<Hash>;
+  getTransferFee: (params: FeeParams) => Promise<BN>;
+  getGiftTransferFee: (params: FeeParams) => Promise<BN>;
+  getExistentialDeposit: () => Promise<BN>;
```

Comment:
Transfer is providing information about existential deposit? Sounds like a vialoation of Single Responsibility principle

Reply:
Moved `getExistentialDeposit` to new Balance service